app/components/search/Filtering.jsx

```diff
@@ -74,8 +74,8 @@ class Filtering extends Component {
               Open now
             </button>
             <div className={"custom-refinement " + (filtersActive ? 'active' : '')}>
-              <EligibilitiesRefinementList attribute="eligibilities" />
-              <CategoriesRefinementList attribute="categories" />
+              <EligibilitiesRefinementList attribute="eligibilities" limit={100} />
+              <CategoriesRefinementList attribute="categories" limit={110} />
```

Comment:
Could you add a comment describing why you have explicitly set the limits? This will help future readers of the code understand why the code is the way it is.